apps/sms/views/conversation/js/conversation.js

```diff
@@ -516,6 +547,11 @@ var ConversationView = {
 
     var prevPanel = args.meta.prev;
 
+    var emailThread = Settings.supportEmailRecipient &&
+      Threads.active.participants.some(Utils.isEmailAddress);
```

Comment:
I'd not do this right now as I'm planning to move ```hasEmailRecipients``` to NewMessageView entirely :)